tests/php/test-amp-image-dimension-extractor.php

```diff
@@ -5,14 +5,6 @@
  * @package AMP
  */
 
-define( 'AMP_IMG_DIMENSION_TEST_INVALID_FILE', dirname( __FILE__ ) . '/assets/not-exists.png' );
```

Comment:
I've refactored the `external-http` group of tests to their own class called `AMP_Image_Dimension_Extract_Download_Test`, which makes it easier to add and remove from test suites.